plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -50,13 +49,6 @@ protected QueryRunner createQueryRunner()
                 TpchTable.getTables());
     }
 
-    @AfterClass(alwaysRun = true)
-    public final void destroy()
-    {
-        mysqlServer.close();
```

Comment:
I see `closeAfterClass` benefits, especially where state management is hard to do without it. That was _the_ reason why we added that method.
But no, it does not improve readability. It makes is _less_ clear that the resources is properly disposed of after the test class.

Reply:
> But no, it does not improve readability.

I don't agree.

Pros of `closeAfterClass`:
 - it is concise. Used when resource is created is like `try-with-resources` you know already that it is going to bel deallocated properly.
 - It is also **correct**. Resources are deallocated in proper order.
 
Cons of `@AfterClass` kind of deallocation: 
 - it is detached from allocation. You need to find a place where object is deallocated. In this commit I removed a case where object was created in one class and destroyed in other. Since it is detached one can even deallocate thing several times.
 - I always need to double check if `alwaysRun` is used. Wasted energy, while with `closeAfterClass` I know it is implemented correctly.
 - more boilerplate, like a need to create field.

Reply:
You missed the fact `@AfterClass` cleanup is standard and globally applicable, whereas `closeAfterClass` is available in these tests only. 

I am not OK making `closeAfterClass` a rule. I am 'whatever' on a change here